Add some text to mark Nextcloud SSO as an experimental feature (#18844)
opf/openproject: docs/release-notes/16-0-0/README.md

Please see [our system admin guide](../../system-admin-guide/integrations/nextcloud/oidc-sso/) to learn how to set up this integration.

> This is the first version of OpenProject to support this new way of integrating Nextcloud and OpenProject. Since this feature was not yet

Reviewer: ```suggestion
This is the first version of OpenProject to support this new integration method with Nextcloud. Since this feature was not yet
```
